understanding/22/target-size-minimum.html

```diff
@@ -57,9 +57,9 @@ <h2>Intent of Target Size (Minimum)</h2>
         <p>The requirement is independent of the zoom factor of the page; when users zoom in the CSS pixel size of elements does not change. This means that authors cannot meet it by claiming that the target will have enough spacing or sufficient size if the user zooms into the page.</p>
 
         <p>While the Success Criterion primarily helps touch users by providing target sizing to prevent accidental triggering of adjacent targets, it is also useful for mouse or pen users. It reduces the chances of erroneous activation due to either a tremor or reduced precision, whether because of reduced fine motor control or input imprecision.</p>
-      
-        <h3>Spacing versus size</h3>
-        <p>As mobile devices generally have internal heuristics to identify which link is closest, spacing can work as effectively as a larger target size. The following illustrate a variety of situations. In the top line of icons, the dimensions of each icon target are 44 by 44 CSS pixels with no space in between. This amply passes this Success Criterion and is actually sufficient to meet the AAA requirement for <a href="../21/target-size-enhanced.html">Target Size (Enhanced)</a>. In the next row, the dimensions of each icon target are 24 by 24 CSS pixels with no space in between, passing this Success Criterion. In the third row, the the same icon targets are only 20 by 20 CSS pixels but they have a 4 CSS pixel space in between, passing this Success Criterion via the spacing exception. In the last row, the the same icon targets are only 20 by 20 CSS pixels and have no space in between, failing this Success Criterion.</p>
+           <h3>Spacing versus size</h3>
+        <p>Spacing, alone and in conjunction with size, can improve user experience. There is less chance of accidentally activating a neighboring target if a target is not immediately adjacent to another. The value of spacing around targets is enhanced by touchscreen devices and user agents, which generally have internal heuristics to identify which link or control is closest to a user's touch interaction.</p>
```

Comment:
perhaps nitpicking but not sure if "the value of spacing" may be misinterpreted by some readers (particularly non-native english speakers) to mean the value/number/magnitude of spacing (i.e. erroneously interpreting this as meaning some touchscreen devices use heuristics to increase spacing).